src/components/PageRouter.js

```diff
@@ -67,12 +66,13 @@ class PageRouter extends React.Component {
     const { previousPath, branches, branch } = this.state
 
     const tools = branch && branch.route.toolbars && branch.route.toolbars(branch.match)
+    const isBreadcrumd = branch && branch.route.breadcrumb !== false
 
     const RenderComponent = branch.route.component
     return (
       <div id='page-router'>
-        <Breadcrumb branches={branches} />
-        {tools || <Toolbar />}
+        {isBreadcrumd && <Breadcrumb branches={branches} />}
+        {tools || null}
```

Comment:
Do you see any reason why this can't just be:
```jsx
   {tools}
```

I'm fairly sure that react can handle a `false` as one of the child elements in the children array.